ironfish/src/memPool/memPool.ts

```diff
@@ -110,6 +110,16 @@ export class MemPool {
     return true
   }
 
+  async getEstimatedFee(): Promise<{ fee: bigint;}> {
```

Comment:
This function doesn't do anything asynchronous so I think we can drop the async and promise.